CHANGELOG.md

```diff
@@ -0,0 +1,28 @@
+## Version 0.1 (September, 2022)
+
+- ENH: Add ability to extract parts of header metadata via 
+  `TableBuilderReader.read_header_metadata()`
+- ENH: Add support for reading files with filters
+- Change default API for reading dataframes to `TableBuilderReader.read_table()`
+  Old API name `TableBuilderReader.read()` has been kept for backwards compatibility
+- ENH: Add support to remove total columns from data via
+  `TableBuilderReader.read_header_metadata()`
+- ENH: Add support to strip "Total" rows/ columns via 
```

Comment:
this is great, I have been doing it by hand so far

Reply:
You mentioned this and I came across a use for it as well, so now it's included.